apps/backend/src/libs/modules/plugin/token-verification/token-verification.plugin.ts

```diff
@@ -0,0 +1,73 @@
+import {
+	FastifyInstance,
+	FastifyPluginOptions,
+	FastifyReply,
+	FastifyRequest,
+	HookHandlerDoneFunction,
+} from "fastify";
+import fp from "fastify-plugin";
+import { JWTPayload, jwtVerify } from "jose";
+
+import { ErrorMessage } from "~/libs/enums/enums.js";
+import { AuthError, HTTPError, ServerError } from "~/libs/modules/http/http.js";
+
+declare module "fastify" {
+	interface FastifyRequest {
+		userPayload?: { id: number } | JWTPayload;
+		verifyJWT: (request: FastifyRequest, reply: FastifyReply) => Promise<void>;
+	}
+}
+
+const verifyJWT = function (
```

Comment:
No need to have this plugin. We will have authorization plugin, which will inject the user to the request. Will be done within #50

Reply:
I don't quite understand then what I need to do. I did as you wrote in the previous proposal for a change, but you said that there should be a plugin. I then just need to get the user id from the request, which will be added when the authorization plugin is implemented, and then get the user from the database?

Reply:
That's right, we need a plugin, but not `verifyJWT` plugin. We need authorization plugin, which will get JWT, decode it, find user by id and inject user into request

Reply:
So, do I need to leave only the code I have in the controller and service now (changing the implementation to use the user from the request) and remove the plugin? Or what?

Reply:
You should discuss with Kemal and agree on who will do which part, because your tasks overlap in some parts. You should work together